_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,4 @@
+{{ site.data.products.db }} clusters support two types of backups:
+
+- Managed-service backups: Cockroach Labs takes automated backups of {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters that are stored in Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page for the type and frequency of backups for [{{ site.data.products.serverless }}](../cockroachcloud/backups-page.html?#backups-page) clusters and [{{ site.data.products.dedicated }}](../cockroachcloud/backups-page.html?filters=dedicated#backups-page) clusters
+- {% if page.name == "run-bulk-operations.md" %} Customer-owned backups {% else %} [Self-service backups](../cockroachcloud/run-bulk-operations.html) {% endif %}: You can take manual backups and store them in your [cloud storage buckets](../{{site.versions["stable"]}}/use-cloud-storage-for-bulk-operations.html) using the [`BACKUP`](../{{site.versions["stable"]}}/backup.html) statement
```

Comment:
On this second bullet point, it looks as though you may have forgotten to update the second instance of "self-service backup" to "customer-owned backups". When I move to the overview page, it still shows "self-service backups".